data/mods/DinoMod/monstergroups/zinosaur.json

```diff
@@ -3,401 +3,307 @@
   },
   {
     "type": "monstergroup",
     "name": "GROUP_zachyrhinosaurus_UPGRADE",
-    "monsters": [
-      { "monster": "mon_sachyrhinosaurus", "weight": 600, "cost_multiplier": 5 },
-      { "monster": "mon_zachyrhinosaurus_brute", "weight": 400, "cost_multiplier": 5 }
-    ]
+    "monsters": [ { "monster": "mon_sachyrhinosaurus", "weight": 600 }, { "monster": "mon_zachyrhinosaurus_brute", "weight": 400 } ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_zentaceratops_UPGRADE",
-    "monsters": [
-      { "monster": "mon_sentaceratops", "weight": 600, "cost_multiplier": 5 },
-      { "monster": "mon_zentaceratops_brute", "weight": 400, "cost_multiplier": 5 }
-    ]
+    "monsters": [ { "monster": "mon_sentaceratops", "weight": 600 }, { "monster": "mon_zentaceratops_brute", "weight": 400 } ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_zosmoceratops_UPGRADE",
-    "monsters": [
-      { "monster": "mon_sosmoceratops", "weight": 600, "cost_multiplier": 5 },
-      { "monster": "mon_zosmoceratops_brute", "weight": 400, "cost_multiplier": 5 }
-    ]
+    "monsters": [ { "monster": "mon_sosmoceratops", "weight": 600 }, { "monster": "mon_zosmoceratops_brute", "weight": 400 } ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_zorosaurus_UPGRADE",
-    "monsters": [
-      { "monster": "mon_sorosaurus", "weight": 600, "cost_multiplier": 5 },
-      { "monster": "mon_zorosaurus_brute", "weight": 400, "cost_multiplier": 5 }
-    ]
+    "monsters": [ { "monster": "mon_sorosaurus", "weight": 600 }, { "monster": "mon_zorosaurus_brute", "weight": 400 } ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_zriceratops_UPGRADE",
-    "monsters": [
-      { "monster": "mon_sriceratops", "weight": 600, "cost_multiplier": 5 },
-      { "monster": "mon_zriceratops_brute", "weight": 400, "cost_multiplier": 5 }
-    ]
+    "monsters": [ { "monster": "mon_sriceratops", "weight": 600 }, { "monster": "mon_zriceratops_brute", "weight": 400 } ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_zyrannosaurus_BRUTE",
-    "monsters": [
-      { "monster": "mon_zyrannosaurus_hulk", "weight": 600, "cost_multiplier": 10 },
-      { "monster": "mon_zyrannosaurus_truck", "weight": 400, "cost_multiplier": 10 }
-    ]
+    "monsters": [ { "monster": "mon_zyrannosaurus_hulk", "weight": 600 }, { "monster": "mon_zyrannosaurus_truck", "weight": 400 } ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_zankylosaurus_BRUTE",
-    "monsters": [
-      { "monster": "mon_zankylosaurus_hulk", "weight": 600, "cost_multiplier": 10 },
-      { "monster": "mon_zankylosaurus_tank", "weight": 400, "cost_multiplier": 10 }
-    ]
+    "monsters": [ { "monster": "mon_zankylosaurus_hulk", "weight": 600 }, { "monster": "mon_zankylosaurus_tank", "weight": 400 } ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_ZOMBIE",
     "monsters": [
-      { "monster": "mon_zombie", "weight": 988 },
```

Comment:
This line removal is the most important piece here

Reply:
I think that was originally to compensate for the 1000 weight total from the previous monstergroup behaviour. Thanks for fixing it!